torch/nn/parallel/distributed.py

```diff
@@ -210,7 +210,8 @@ class DistributedDataParallel(Module):
     """
     def __init__(self, module, device_ids=None,
                  output_device=None, dim=0, broadcast_buffers=True,
-                 process_group=None, bucket_cap_mb=25,
+                 process_group=None,
+                 bucket_cap_mb=25,
```

Comment:
I tried to use exported constant from C++, but it failed on windows build. windows "import torch" will import torch.nn.parallel modules that include importing DistributedDataParallel. but windows do not have distributed package, and will complain exported constant is not available. 

I also tried to import DistributedDataParallel only if distributed.is_available() in torch.nn._init_, but that break a lot of other projects. 

I think it should be fine to let this constant in both python and C++? as they are defaults for different constructors. one is for DistributedDataParallel, one is for pybinded reducer. DistributedDataParallel and reducer are usually used together, we always pass bucket_cap_mb to reducer from DistributedDataParallel. 

How do you think?

Reply:
I see, yes, in this case, let's keep it this way. This might be another reason that we should move DDP to `torch.distributed`